lib/libc/stdio/lib_sscanf.c

```diff
@@ -340,13 +346,20 @@ int vsscanf(FAR const char *buf, FAR const char *fmt, va_list ap)
 	FAR const char *bufstart;
 	FAR char *tv;
 	FAR const char *tc;
-	bool lflag;
+	int modifier;
 	bool noassign;
 	int count;
 	int width;
 	int fwidth;
 	int base = 10;
```

Comment:
in my opinion, It seems better to add a definition for 10.

Reply:
This is not related with this PR. Let's do that later with new PR.

Reply:
ok, i see.